pkg/volume/csi/csi_mounter_test.go

```diff
@@ -699,65 +695,4 @@ func TestUnmounterTeardown(t *testing.T) {
 	if _, ok := pubs[csiUnmounter.volumeID]; ok {
 		t.Error("csi server may not have received NodeUnpublishVolume call")
 	}
-
-}
-
-func TestSaveVolumeData(t *testing.T) {
```

Comment:
Why is this test case removed?

Reply:
@msau42 Not, removed. Just moved to file csi_util_test.go since the function Save/LoadVolumeData are located in cs_util.go